suite/tests/api/dis-x64.expect

```diff
@@ -46,7 +46,8 @@
 +0x0056   8f 40 ff             pop    qword ptr [rax-0x01]
 +0x0059   72 e3                jb     0x000000001000003e
 +0x005b   d7                   xlatb
-+0x005c   62 83 84 d1 f7 99...??  <INVALID>
++0x005c   62 83 84 d1 f7 99 f4...??  <INVALID>
+ 6d 97 8e
```

Comment:
In 64-bit mode these are invalid instructions (invalid EVEX instructions). Both objdump and XED decode as such, behaving differently (in terms of length). In our case, we decode as INVALID not because we know, but because we are missing full decoder support. The skip length is in line with xed and objdump but there is no 'right' length in this case. Resolving the comment here, re-open if necessary